test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,481 @@
+                .transfer(AddressZeroSigner.address, standardAmount)
+                .then((tx) => tx.wait());
+            // send some gas token
+            iexecAdmin
+                .sendTransaction({
+                    to: AddressZeroSigner.address,
+                    value: AmountWithDecimals(BigNumber.from(100_000)),
+                })
+                .then((tx) => tx.wait());
+
+            await rlcInstance
+                .connect(AddressZeroSigner)
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            await expect(
+                iexecPoco.connect(AddressZeroSigner).deposit(standardAmount),
+            ).to.be.revertedWith('ERC20: mint to the zero address');
+        });
+    });
+
+    describe('Deposit for', () => {
+        it('Should deposit tokens for another account', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            const depositForParams = {
+                amount: standardAmount,
+                target: accountB.address,
+            };
+            const depositForArgs = Object.values(depositForParams) as [BigNumber, string];
+
+            expect(await iexecPocoAsAccountA.callStatic.depositFor(...depositForArgs)).to.be.true;
+            await expect(iexecPocoAsAccountA.depositFor(...depositForArgs))
+                .to.changeTokenBalances(
+                    rlcInstance,
+                    [accountA, iexecPoco],
+                    [-depositForParams.amount, depositForParams.amount],
+                )
+                .to.emit(rlcInstance, 'Transfer')
+                .withArgs(accountA.address, iexecPoco, depositForParams.amount)
+                .to.changeTokenBalances(iexecPoco, [depositForParams.target], [standardAmount])
+                .to.emit(iexecPoco, 'Transfer')
+                .withArgs(AddressZero, depositForParams.target, depositForParams.amount);
+        });
+        it('Should not deposit tokens for zero address', async () => {
+            await rlcInstanceAsAccountA
+                .approve(iexecPoco.address, standardAmount)
+                .then((tx) => tx.wait());
+
+            const depositForParams = {
+                amount: standardAmount,
+                target: AddressZero,
+            };
+            const depositForArgs = Object.values(depositForParams) as [BigNumber, string];
+            await expect(iexecPocoAsAccountA.depositFor(...depositForArgs)).to.be.revertedWith(
+                'ERC20: mint to the zero address',
+            );
+        });
+    });
+
+    describe('Deposit for array', () => {
+        it('Should deposit tokens for multiple accounts', async () => {
+            const depositForArrayParams = {
+                amounts: [standardAmount, standardAmount.mul(2)],
+                targets: [iexecAdmin.address, accountB.address],
```

Comment:
```suggestion
                targets: [accountB.address, accountC.address],
```
From A to (B & C), no need for special admin rights

Reply:
[introduce account C and clean account B](https://github.com/iExecBlockchainComputing/PoCo/pull/141/commits/478f09f557b3d7215ce2ad22dcf69d86da02cf6f)